src/constants/common.ts

```diff
@@ -8,7 +8,7 @@ export const isDev = import.meta.env.DEV
 /**
  * @source
  */
-export const includeTestNets = env.PUBLIC_INCLUDE_TESTNETS
+export const includeTestnets = env.PUBLIC_INCLUDE_TESTNETS
```

Comment:
Default to `true` or `false` to improve the type here until zod is used to `parseEnv`.

Reply:
@paterson1 it's using zod, and is set to a default https://github.com/BootNodeDev/dAppBooster/blob/b305e7205dd8c1f4404156f2dffa2ecc10bc3961/src/env.ts#L45